docs/testing.md

```diff
@@ -23,4 +23,4 @@ Since fuzzing involves generating inputs at time intervals, you explicitly run t
 Watch the output, and view any errors that may arise. When you are satisfied the tests confirm your code working as expected, you can exit the test. 
 
 eg. 
-` go test -fuzz=FuzzNewVersionManager -v`
+` go test -fuzz=FuzzVersionManager_Delete -v`
```

Comment:
Do you think it would be a good idea to update GitHub actions with fuzzing? Alternatively, we could create a GitHub action that PRs don't depend on that performs fuzzing.

Reply:
Hi! Is this question about whether these tests will run automatically? 

Currently, the fuzz tests will need to be run manually, as described in the doc.

Reply:
Makes sense, thank you!